src/main/java/com/internship/backend/filter/RequestValidationBeforeFilter.java

```diff
@@ -0,0 +1,51 @@
+package com.internship.backend.filter;
+import jakarta.servlet.*;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.springframework.security.authentication.BadCredentialsException;
+import org.springframework.util.StringUtils;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+
+
+public class RequestValidationBeforeFilter  implements Filter {
```

Comment:
Could javadoc be added to this class explaining its general purpose, like which validation performs?

Reply:
I tried to make a custom filter. I parse the HttpHeader, and check if my email contains the word "simon".
For example, when I want to login and my email contains the word "simon", it will bad request.
I followed this tutorial: https://ppb.udemy.com/course/spring-security-zero-to-master/learn/lecture/23181292#overview